os/binfmt/libelf/libelf_ctors.c

```diff
@@ -175,14 +175,22 @@ int elf_loadctors(FAR struct elf_loadinfo_s *loadinfo)
 		if ((shdr->sh_flags & SHF_ALLOC) == 0) {
 			/* Allocate memory to hold a copy of the .ctor section */
 
+			loadinfo->ctoralloc = (uint32_t)kmm_malloc(ctorsize);
+			if (!loadinfo->ctoralloc) {
+				berr("Failed to allocate memory for .ctors\n");
```

Comment:
Is ".ctors" a typo here?

Reply:
We are allocating memory for .ctors section. So it's fine.